templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig

```diff
@@ -723,6 +725,17 @@
                                 </div>
                             {% endif %}
 
+                            <addon-wrapper
```

Comment:
Nitpick (not to be implemented in this PR): `addon.additional.field` is a _very_ generic key. Also: does the actual hook name need to contain "addon" in it? as the whole thing is in `<addon-wrapper />`, i think it is clear what we're looking at. I'd suggest to name it more specific to this view and section, like `basicSettings.nameAddress.additionalFields`.